native/src/components/MapView.tsx

```diff
@@ -94,31 +109,46 @@ const MapView = ({ boundingBox, featureCollection, currentFeature }: MapViewProp
       ? 'location-searching'
       : 'location-disabled'
 
-  const onPress = useCallback(async (pressedLocation: Feature) => {
-    if (!mapRef?.current || !cameraRef?.current || !pressedLocation.properties) {
-      return
-    }
-    const featureCollection = await mapRef.current.queryRenderedFeaturesAtPoint(
-      [pressedLocation.properties.screenPointX, pressedLocation.properties.screenPointY],
-      undefined,
-      [featureLayerId]
-    )
-    const feature = featureCollection?.features?.find((it): it is Feature<Point> => it.geometry.type === 'Point')
-    if (feature) {
-      setSelectedFeature(feature)
-      const {
-        geometry: { coordinates }
-      } = feature
-      cameraRef.current.flyTo(coordinates)
-    } else {
-      setSelectedFeature(null)
-    }
-  }, [])
+  // set the user location coordinate once to calculate distance for all pois
+  const onLocationUpdate = (location: MapboxGL.Location): void => {
+    !userLocation && setUserLocation([location.coords.longitude, location.coords.latitude])
+  }
+
+  const onPress = useCallback(
+    async (pressedLocation: Feature) => {
+      if (!mapRef?.current || !cameraRef?.current || !pressedLocation.properties) {
+        return
+      }
+      const featureCollection = await mapRef.current.queryRenderedFeaturesAtPoint(
+        [pressedLocation.properties.screenPointX, pressedLocation.properties.screenPointY],
+        undefined,
+        [featureLayerId]
+      )
+      const feature = featureCollection?.features?.find((it): it is Feature<Point> => it.geometry.type === 'Point')
+      if (feature) {
+        const {
+          geometry: { coordinates }
+        } = feature
+        setSelectedFeature(feature)
+
+        cameraRef.current.flyTo(coordinates)
+      } else {
+        setSelectedFeature(null)
+      }
+    },
+    [setSelectedFeature]
+  )
 
   return (
     <MapContainer>
-      <StyledMap styleJSON={mapConfig.styleJSON} zoomEnabled onPress={onPress} ref={mapRef}>
-        <MapboxGL.UserLocation visible={locationPermissionGranted} />
+      <StyledMap
+        styleJSON={mapConfig.styleJSON}
+        zoomEnabled
+        onPress={onPress}
+        ref={mapRef}
+        attributionEnabled={false}
+        logoEnabled={false}>
```

Comment:
Are we allowed to disable these? https://docs.mapbox.com/help/glossary/attribution/
> You must properly attribute and link Mapbox and OpenStreetMap when using the Mapbox Streets tileset.

Reply:
but we don't use mapbox but maplibre

Reply:
We still use OSM though. But yes, was just a question whether you know if we have to attribute those.